.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: "Label PRs"
+on:
+- pull_request_target
+
+jobs:
+  create_release:
```

Comment:
is this create_release a reserved keyword for release or is it something used to run the actions?

Reply:
Sorry, missed this in the copy-paste; I believe this label is just a name for the job itself, it can be anything. What do you think of `apply_pr_labels`?

Reply:
sounds good! nit: You can just say it `apply_labels`

Reply:
Awesome! Updated job name to suggested text